Habitica/src/test/java/com/habitrpg/android/habitica/api/TagAPITests.java

```diff
@@ -25,6 +30,48 @@ public void shouldCreateTag() {
         testSubscriber.assertNoErrors();
         testSubscriber.assertCompleted();
         testSubscriber.assertValueCount(1);
+
+    }
+
+    @Test
+    public void shouldUpdateTag() {
+        TestSubscriber<Tag> testSubscriber = new TestSubscriber<>();
+        TestSubscriber<HabitRPGUser> userSubscriber = new TestSubscriber<>();
+
+        Tag t = new Tag();
+        String newname = "BAR";
+        t.setId(String.valueOf(UUID.randomUUID()));
+        t.setName(newname);
+
+        //Get the test user so we can obtain their tags
+        apiHelper.apiService.getUser().subscribe(userSubscriber);
+        userSubscriber.assertNoErrors();
+        userSubscriber.assertCompleted();
+        List<HabitRPGUser> users = userSubscriber.getOnNextEvents();
```

Comment:
Could you move this to something like a `getUser` method on the `BaseAPITests`, since it is something that is used in different places?